docs/flashbots-protect/rpc/fast.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Flashbots Protect Fast
+---
+
+Flashbots Protect offers two ways to send your transaction: Flashbots Protect and Flashbots Protect Fast. Standard Flashbots Protect uses Flashbots bundles, whereas Flashbots Protect Fast uses a new transaction type called "private transactions." Flashbots Protect Fast transactions are more likely to be included fast, but they don't have protection against reverts and don't get priority at the top of the block.
```

Comment:
> Flashbots Protect Fast transactions are more likely to be included fast

Would change to 
"Flashbots Protect Fast Mode transactions are more likely to be included sooner"

Reply:
fyi-We internally call this as Fast preference. Going forward we might have different preferences such as Auction etc

Reply:
I feel that "Flashbots Protect Fast transactions" sounds a bit better than "Flashbots Protect Fast Mode transactions"